clang/lib/CIR/CodeGen/CIRGenBuiltinX86.cpp

```diff
@@ -43,7 +43,56 @@ mlir::Value CIRGenFunction::emitX86BuiltinExpr(unsigned builtinID,
   // Find out if any arguments are required to be integer constant expressions.
   assert(!cir::MissingFeatures::handleBuiltinICEArguments());
 
-  switch (builtinID) {
+  llvm::SmallVector<mlir::Value, 4> Ops;
```

Comment:
```suggestion
  llvm::SmallVector<mlir::Value> Ops;
```
Unless there is some concrete reason for choosing the initial vector size, it is better to accept the default.